lib/napa/deprecations/application_api.rb

```diff
@@ -1,8 +1,10 @@
 module Napa
   class Deprecations
+    LINK = 'https://github.com/bellycard/napa/blob/master/lib/napa/generators/templates/scaffold/app/apis/application_api.rb'
```

Comment:
@darbyfrey this file doesn't seem to be alive anymore. Should remove this or has that file been moved somewhere else?